packages/plugin-http-client/src/HttpClient.ts

```diff
@@ -0,0 +1,119 @@
+import type {
+  Dependencies,
+  HttpAgent,
+  HttpAgentRequestOptions,
+  UnknownParameters,
+} from '@ima/core';
+
+declare module '@ima/core' {
+  interface OCAliasMap {
+    $DefaultProcessors: Processor[];
+  }
+}
+
+import { Processor, Operation, ProcessorParams } from './Processor';
+
+export const OPTION_TRANSFORM_PROCESSORS = 'transformProcessors';
+
+export type HttpClientRequestOptions = {
+  [OPTION_TRANSFORM_PROCESSORS]?: (processor: Processor[]) => Processor[];
+} & HttpAgentRequestOptions;
+
+export type HttpClientRequestMethod =
+  | 'get'
```

Comment:
Není tady náhodou ta pajpa navíc?

Reply:
Přidává ji tam lint - je tam jen, aby se to lepe četlo - je tam k ničemu, ale ničemu nevadí.